homework/index.html

```diff
@@ -0,0 +1,233 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css" integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==" crossorigin="anonymous" />
+    <title>Adrian Ursu</title>
+</head>
+<body>
+    <header>
+        <section>
+            <h1>Adrian Ursu</h1>
+            <img src="img/photo.jpeg" alt="Photo of Adrian"      width="200">
+            <nav>
+                <a href="#about"><strong>About</strong></a>
+                <a href="#skills"><strong>Skills</strong></a>
+                <a href="#projects"><strong>My Projects</strong></a>
+                <a href="#study"><strong>Study</strong></a>
+                <a href="#work"><strong>Work</strong></a>
+                <a href="#contacts"><strong>Contacts</strong></a>
+                <ul>
+                    <li><a href="#proposal"><strong>Send Proposal</strong></a></li>
+                </ul>
+            </nav>
+        </section>
+    </header>
+            <hr>
+    <main>
+        <div id="about">
+            <section>
+                <h2>ABOUT</h2>
+                <p><strong>Lorem ipsum</strong>, dolor sit amet consectetur adipisicing elit. Nesciunt qui, nulla quam minus sit voluptatum cupiditate eos molestiae atque neque deserunt eum maiores eligendi expedita.</p>
+                <p><strong>Lorem</strong>, ipsum dolor sit amet consectetur adipisicing elit. Nostrum rem quasi omnis modi iusto blanditiis laboriosam quod velit exercitationem odit, unde nesciunt cum nemo officiis corporis id minus neque autem reprehenderit ad quaerat, odio iure!</p>
+                <p><strong>Lorem ipsum dolor sit</strong>, amet consectetur adipisicing elit. Dicta, vel itaque tempore eos deserunt soluta. Delectus iusto quam laboriosam atque facilis aliquid aut quo neque quisquam. Omnis facere ea quo, eaque deleniti similique, itaque dolor possimus, autem eos consequatur expedita provident iure inventore aliquam voluptatem magni natus dicta odit voluptate.</p>
+            </section>
+        </div> <!-- About -->
+
+            <hr>
+
+        <div id="skills">
+            <section>
+                <h2>SKILLS</h2>
+                <ul>
+                    <li>Web Development</li>
+                         <ol>

```

Comment:
ul tag has some content limitations
![image](https://user-images.githubusercontent.com/26486750/121858448-4c136b80-ccff-11eb-8bde-07cafa147d25.png)

In your case this **ol** tag should be part of **li** tag like this
```html
<li>
    Web Development
    <ol>
        <li>HTML</li>
        <li>CSS</li>
        <li>JavaScript</li>
        <li>PHP</li>
        <li>Laravel</li>
        </ol>
</li>
````
**ol** tag is not a child of **ul**

Reply:
done